.travis.yml

```diff
@@ -30,7 +30,7 @@ install:
   - conda activate test-environment
   - conda env update -n test-environment --quiet --file conda_environment.yml
   - conda install pip
-  - conda install -c conda-forge iris
+  - conda install -c conda-forge "iris>=3.1,!=3.2.0.post0"
```

Comment:
we can actually just delete this line. line 31 already installs iris from the conda_environment file.

Reply:
> we can actually just delete this line. line 31 already installs iris from the conda_environment file.

Okay, I was wondering about this.

BUT: I think the `conda env update` command comes with it's own issues. At least at some point, it upgraded my Python version (because it says `python>=3.6`), which is not what we want, do we? So I wonder in general how reliable this recipe really is.

Reply:
For reference, this is what I found is a reliable way to install the test environment:
```
conda create -n testenv -c conda-forge python=3.x
conda activate testenv
conda install -c conda-forge "iris>=3.1,!=3.2.0.post0"
pip install -e .  # source install from the cloned git repo
pip install [...]  # whatever else is needed to run the tests (coverage, pytest, ...)
```
This can of course be adapted again to some of the helpful extras of the `.travis` recipe and also using some environment file.